ships/project-look-and-feel.md

```diff
@@ -0,0 +1,273 @@
+Agreeing to a standard structure will help community members make contributions between
+repositories and help new community members navigate the codebase.
+
+### Goals
+
+1. Outline a standard structure for Shipwright controllers.
+2. Define the toolchain that will be used to build and deploy container images.
+
+### Non-Goals
+
+1. Define a standard structure for the shp cli. The cli is built on different libraries and
+   should not be beholden to the KubeBuilder structure.
+2. Define a standard structure for the website. The website is built with [Hugo](https://gohugo.io/)
+   and should conform to this framework's structure.
+3. Centralize tooling for testing with GitHub actions.
+4. Centralize release tooling.
+5. Prescribe a standard structure or format for documentation.
+
+## Proposal
+
+### Project Structure
+
+Controller repositories should standardize on the following form, based on the KubeBuilder v3
+project structure:
+
+```
+.github/
+api/
+client/
+cmd/
+config/
+controllers/
+docs/
+hack/
+pkg/
+test/
+tools/
+vendor/
+.gitignore
+CONTRIBUTING.md
+LICENSE
+go.mod
+go.sum
+main.go
+Makefile
+PROJECT
+README.md
+```
+
+The following directories and files are derived from the Kubebuilder v3 format:
+
+- `api/`: Expected when generating APIs and webhooks
+- `config/`: Always expected
+- `controllers/`: Expected when generating controllers
+- `hack/`: Expected when generating deep copy functions
+- `.gitignore`
+- `LICENSE`
+- `go.mod`
+- `go.sum`
+- `main.go` : Required by kubebuilder
+- `Makefile`
+- `PROJECT`
+
+Keeping this structure ensures that Kubebuilder's command line utilites can be used to simplify
+future development (for example - adding new APIs, controllers, and webhooks).
+
+The following directories are extensions based on our own experiences with Shipwright:
+
+- `.github/`: Configuration for GitHub and GitHub Actions.
+- `client/`: Generated client code. This is optional, and will hold generated go client code.
+- `docs/`: In-tree documentation.
+- `pkg/`: Business logic that can be shared between controllers and/or tested outside of a
+  reconciliation loop.
+- `test/`: Test specifications for integration and end-to-end testing.
+- `tools/`: Additional tooling that is imported with go modules. This an informal [project layout standard](https://github.com/golang-standards/project-layout#tools).
+- `vendor/`: Vendor tree for go code.
+
+### Tooling
+
+Most of the tooling controllers use will be based on the KubeBuilder v3 toolkit.
+These are the baseline tools used to generate code and run integration tests, and should be
+installed locally with Makefile targets:
+
+- Kustomize
+- Controller-gen
+- Envtest
+
+Shipwright controller projects should also standardize on [ko](https://github.com/google/ko) for
+building golang-based container images.
+Ko provides critical features such as reproducible builds and automatic generation of Software
+Bills of Material (SBOMs).
```

Comment:
The first thing I think about as an advantage of ko is actually the easy way of building cross-platform images. :-)